src/main.js

```diff
@@ -325,10 +349,20 @@ app.on('ready', () => {
       catch((err) => console.log('An error occurred: ', err));
   }
 
+  // Protocol handler for win32
+  if (process.platform === 'win32') {
+    // Keep only command line / deep linked arguments
+    if (Array.isArray(process.argv.slice(1)) && process.argv.slice(1).length > 0) {
```

Comment:
You should also handle the case, command line includes [`--squirrel-*` options](https://github.com/electron-archive/grunt-electron-installer#handling-squirrel-events). In that case, the app should run as an installer/uninstaller. So please do not anything.